wiki/Multi/en.md

```diff
@@ -0,0 +1,422 @@
+**Score v2** | Player with **highest _standardised_ score** wins.
+
+If Team Mode was set to Tag Coop/Tag Team Vs, a combo colour section will be shown which allows fixed combo colour adjustment when it is the player's turn.
+
+Lastly, there are two buttons coloured brownish-orange Leave Match at the left and blue Ready!/Not Ready/Start Game!/Force Start Game! button at the right respectively.
+The Leave Match button is self-explanatory; leave the Match Setup and return to the Lobby.
+
+For the blue button, it can change depending on action taken and host/player privilege.
+The different effects of the blue button can be seen below.
+
+Title | Description
+:---:|:---
+**Ready!**                   | Click to enter Ready state (green state). Button will change to Not Ready for players, either Start Game! or Force Start Game! for host if there are players in the Match Setup, Not Ready otherwise.
+**Not Ready**                | Click to return to Not Ready state (white state). Button will change to Ready!
+**Start Game!**              | Host-only button; starts the match. Appears with a distinctive ring when all players in the Match Setup in Ready! state (green state) including the host. Will switch to Force Start Game! when at least one player pressed the Not Ready button.
+**Force Start Game! (x/\#)** | Host-only button; forcefully ready all players with the beatmap and starts the match. Appears when host press the Ready! button but not all players in the Match Setup pressed the Ready! button (where x is the number of players in Ready! state, and \# is the total players in the Match Setup).
+
+![match history single][Multi_mh_single image]
+
+_Example of Match History (Single)._
+
+![match history team][Multi_mh_team image]
+
+_Example of Match History (Team)._
+
+#### Player box colour
+
+<!-- A player box reference here? -->
+<!-- Special player box note: Skipped (has its own special callout), Failed (red text), and Quit (red text with [Quit] appended) -->
+
+Status/Colour | Description
+:---:|:---
+**Normal/Blue** | **Player has _more_ than half of health**, usually at full. The concentration of blue colour will grow brighter if player's healthbar increasing to full and bleed to red by losing the lifebar more than half the healthbar length.
+**Danger/Purple-Red** | **Player has _less_ than half of health**. The concentration of red colour will grow brighter if player's healthbar decreasing to empty and diminished to blue by filling the lifebar more than half the healthbar length.
+**Failed/Gray** | **Player unable to reserve the final sliver of health/no health left in the healthbar**. Player can continue playing and if [SD][Sudden Death wikilink]/[PF][Perfect wikilink] was not used, player can be "revived" from the failed status by refilling the lifebar completely. Also, score will not be contributed towards the team score at this state. Text colour will be changed from white to red.
```

Comment:
- I would heavily suggest on not using abbreviations for the game mods with the way how this page is written as of now
  - I mean, you haven't even noted&mdash;with round brackets&mdash;on what they mean
    - e.g. `[SD][Sudden Death wikilink] (Sudden Death)`; if you did this, you should be ok from that point downwards